TASVideos/Pages/RssFeeds/Wiki.cshtml

```diff
@@ -20,8 +20,8 @@
 				<rss-link>@url</rss-link>
 				<description>@edit.RevisionMessage</description>
 				<comments>@(Settings.BaseUrl)/Forum/</comments>
-				<guid>@url</guid>
-				<pubDate>@edit.PubDate.ToString("u", CultureInfo.InvariantCulture)</pubDate>
+				<guid>@url?revision=@edit.RevisionId</guid>
```

Comment:
This is a decision. Just pointing it out.  I'll trust it makes more sense

Reply:
As noted in the commit message, these need to be unique within the file or it [fails validation](https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Ftasvideos.org%2FWiki.rss). That doesn't seem to be explicit in the spec, but [it does effectively say](https://www.rssboard.org/rss-specification#ltguidgtSubelementOfLtitemgt) that if the server doesn't ensure uniqueness then clients won't cache correctly.

The spec also notes [guid != link](https://www.rssboard.org/rss-specification#comments), if that was your concern with the revised value.

Reply:
Its actually more than an alias, it IS the name of the route, no redirect occurs when somone goes to /submissions.rss